examples/lock-app/efr32/src/LockManager.cpp

```diff
@@ -104,6 +104,17 @@ bool LockManager::IsValidUserIndex(uint16_t userIndex)
     return (userIndex < kMaxUsers);
 }
 
+bool LockManager::IsCredentialIndexGreaterThanZero(uint16_t credentialIndex, DlCredentialType type)
```

Comment:
The name seems off. it seems to say 'is the credential index greater than zero' which to me translates to 'credentialIndex > 0'.

Please name this something better ... maybe "IsValidCredentialIndex" or something like that.

Reply:
Yeah, but there's already a function called IsValidCredentialIndex for the upper bound :(

I could change it to isValidCredentialIndexLowerBound, and isValidCredentialIndexUpperBound?